webui/react/src/hooks/useModal/JupyterLab/useModalJupyterLab.tsx

```diff
@@ -360,7 +360,7 @@ const JupyterLabForm: React.FC<FormProps> = ({ updateFields, fields }: FormProps
             <InputNumber
               defaultValue={fields.slots !== undefined ? fields.slots : DEFAULT_SLOT_COUNT}
               max={resourceInfo.maxSlots === -1 ? Number.MAX_SAFE_INTEGER : resourceInfo.maxSlots}
-              min={resourceInfo.hasAux ? 0 : 1}
```

Comment:
is this because hasAux is inaccurate? if not what's the issue here. 
I should re-read what hasAux advertises. but if it means that the RP has no support for auxiliary tasks then this won't be able to run 🤔 maybe it's still fine to allow this

Reply:
is there a problem description or jira?

Reply:
So the main issue is that kubernetes doesn't have resource pools, so it's forcing users to use a slot in kubernetes for notebooks. Also on latest-master, if you select the compute pool, it locks you into using at least one slot (which might be the correct behavior, but shouldn't we still allow users to run zero slot tasks?)

Reply:
@NicholasBlaskey mentioned that the problem isn't with hasAux, so this might not be the correct solution.

Reply:
ah okay, looks like the issue was not with `min`, but that we were restoring the default when `slots` input was `0`

@hamidzr you were right, that logic is fine. nicholas and eric were reporting an issue were they couldnt give a notebook zero slots.

Reply:
[this change](https://github.com/determined-ai/determined/pull/4582/commits/4484471792da857951494d50205574846d481194) would have the described behavior 🚀